### ISSUE TITLE
"Graphics Devices" limits the number of GPUs shown

### ISSUE THREAD

**Issue by vincentqb** [bvaisvil/zenith#160]
The list of "Graphics Devices" is visually limited to a few rows (e.g. "Total" and 5 GPUs out of 8).

Using the up and down arrows moves the arrow beyond what is shown in the list but doesn't scroll it. The (hidden) movement of the arrow is consistent with having the correct number of GPUs (e.g. 8).

**Comment by bvaisvil** [comment]
Thanks for the report, I have to admit to not testing beyond 2. If you expand the the section (tab to highlight the graphics section and "e" to expand the section) does it show all of the GPUs?

**Comment by vincentqb** [comment]
Yup, pressing "e" a few times does expand to see the correct full list (and pressing more reveals empty rows).

**Comment by bvaisvil** [comment]
Closing as I think this isn't a bug but depends on the size that pane is set to.
